static/admin/config.yml

```diff
@@ -161,6 +161,7 @@ collections:
                     fields:
                       [
                         { label: Image, name: image, widget: image },
+                        { label: title, name: title, widget: text },
```

Comment:
Label should be with Capital Case, shouldn't it?

Reply:
Yes. Fixed it.